src/annotator/anchoring/test/types-test.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+var types = require('../types');
+
+var TextQuoteAnchor = types.TextQuoteAnchor;
+var TextPositionAnchor = types.TextPositionAnchor;
+
+// These are primarily basic API tests for the anchoring classes. Tests for
+// anchoring a variety of HTML and PDF content exist in `html-test` and
+// `pdf-test`.
+describe('Anchoring classes', function () {
```

Comment:
Wondering if `describe('types.coffee')` (or maybe just `describe('types')`) would be a more precise way of putting this? Or else rename `types.coffee` to `anchoring.coffee`. But there's a mismatch between the two names "types" and "anchoring" which I think makes it less immediately clear, from looking at for example the output of a failed test, what code was being tested.